### PR TITLE
Tutorial : Downsampling and Unlimited Metric Retention for Prometheus.

### PR DESCRIPTION
**Pull request by soniasingla**
Signed-off-by: soniasingla <soniasingla.1812@gmail.com>

This PR aims to add Katacoda tutorial for the course `Downsampling and Unlimited Metric Retention for Prometheus.`

Making the tutorial interactive with Katacoda, @bwplotka and @povilasv 💜

### CODE REVIEW

**Comment by bwplotka** [comment]
Can you post link to the preview of this? (: :hugs:

**Comment by soniasingla** [comment]
https://katacoda.com/soniasingla/ preview 🤗

File: tutorials/katacoda/thanos/2-lts/intro.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
-# In progress
+[Thanos](thanos.io) is a set of components that can be composed into a highly available metric system with unlimited storage capacity. It can be added seamlessly on top of existing Prometheus deployments.
 
-🚧 This tutorial is in progress. 🚧
+Thanos allows users to aggregate Prometheus data natively by directly querying the Prometheus API, efficiently compact it, and most importantly, de-duplicate data.
```
Comment (povilasv) [inline tutorials/katacoda/thanos/2-lts/intro.md:3]: I feel like this sentence doesn't fit our tutorial as it's about unlimited storage retentation, not connecting Prometheis together

File: tutorials/katacoda/thanos/2-lts/step1.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+# Step 1 - Start initial Prometheus servers
+
+In this tutorial, we will be using three Prometheus Servers.
+
+1. We have one Prometheus server in some eu1 cluster.
```
Comment (povilasv) [inline tutorials/katacoda/thanos/2-lts/step1.md:5]:
```suggestion
1. We have one Prometheus server in eu1 region.
```

File: tutorials/katacoda/thanos/2-lts/step1.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+2. We have 2 replica Prometheus servers in some us1 cluster that scrapes the same targets.
```
Comment (povilasv) [inline tutorials/katacoda/thanos/2-lts/step1.md:6]:
```suggestion
2. We have 2 Prometheus servers scraping the same target in us1 region,
```

File: tutorials/katacoda/thanos/2-lts/step1.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    quay.io/prometheus/prometheus:v2.14.0 \
+    --config.file=/etc/prometheus/prometheus.yml \
+    --storage.tsdb.path=/prometheus \
+    --web.listen-address=:9090 \
+    --web.external-url=https://[[HOST_SUBDOMAIN]]-9090-[[KATACODA_HOST]].environments.katacoda.com \
+    --web.enable-lifecycle \
+    --web.enable-admin-api && echo "Prometheus EU1 started!"
+```{{execute}}
+
+NOTE: We are using the latest Prometheus image so we can take profit from the latest remote read protocol.
```
Comment (povilasv) [inline tutorials/katacoda/thanos/2-lts/step1.md:89]: This note will go out of date soon, so lets remove it

**Comment by povilasv** [comment]
I think most of the text is copied from previous tutorial, maybe we can change it a bit and make it simpler. We don't care about deduplication or about connecting Prometheus together. 

step1: So let's have 2 Prometheis in different regions
step 2 attach sidecars with config to push to s3 (minio), 
step 3 : deploy Thanos Store gateway and querier, query some metrics from S3.
step 4: attach compactor and talk about downsampling?

**Comment by soniasingla** [comment]
Seems there is some problem with Netlify 🤔

File: tutorials/katacoda/thanos/2-lts/intro.md (hunk 1 above)
Comment (soniasingla) [reply tutorials/katacoda/thanos/2-lts/intro.md:3]: Thanks for the review, I'll update it 🤗

**Pull request closed without merging**